src/Endpoints/Subscriptions.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace PagarMe\Endpoints;
+
+use PagarMe\Client;
+use PagarMe\Routes;
+use PagarMe\Endpoints\Endpoint;
+
+class Subscriptions extends Endpoint
+{
+    /**
+     * @param array $payload
+     *
+     * @return \ArrayObject
+     */
+    public function create(array $payload)
+    {
+        return $this->client->request(
+            self::POST,
+            Routes::subscriptions()->base(),
+            ['json' => $payload]
+        );
+    }
+
+    /**
+     * @param array $payload
+     *
+     * @return \ArrayObject
+     */
+    public function createSplit(array $payload)
```

Comment:
I didn't understand the purpose of this function, can you explain?

Reply:
I think this function can cause a misundertood to the user. Reading it name, I  think i have to use it like:

```php
$pagarme->subscriptions()->createSplit([
    'split_rules' => [
         [
             'recipient_id' => 're_ci7nheu0m0006n016o5sglg9t',
             'charge_processing_fee' => true,
             'liable' => true,
             'percentage' => '60',
         ],
         [
             'recipient_id' => 're_ci7nhf1ay0007n016wd5t22nl',
             'charge_processing_fee' => true,
             'liable' => false,
             'percentage' => '40',
         ]
    ],
]);
```

Reply:
Concordo com o @murilohns nesse ponto. Eu acho que o sdk deveria refletir o endpoints da API como eles são. Não vejo muito ganho na função `createSplit` dado que ela faz a mesma coisa que a `create`.

Reply:
Reading your previous comments, now I agree with @murilohns. Also I read our doc and I saw that the user should add `split_rules` to payload to create splitted transactions.

I get your point @devdrops about the "alias" but `createSplit` doesn't enforce a contract to inform `split_rules` or prevent any errors from the developer that will use our SDK. So, given that there's no difference I think that it doesn't add value and may cause a misunderstanding as @murilohns said.

Reply:
@murilohns @leonampd @piiih thanks for the feedback!